### DISCUSSION TITLE
This user account is blocked. Vercel

### DISCUSSION THREAD

**Discussion by Damarwendha** [vercel/community#4061] [Account Issue] [answered]
### Question

When I try to sign up and log in, I get an error message saying, "This user account is blocked." I've already contacted email support, but I haven't received any reply yet.

**Comment by amyegan** [comment, marked as answer]
Hey @Damarwendha. We can't safely discuss account information in public because of security concerns. You did the right thing by sending that email. 

Tickets are processed by the order in which they are received. You can learn more about response times and how Vercel Support works at https://vercel.com/guides/vercel-support-queue-time
